js/bybit.js

```diff
@@ -1637,7 +1638,7 @@ module.exports = class bybit extends Exchange {
         if (symbols !== undefined) {
             const symbol = this.safeValue (symbols, 0);
             market = this.market (symbol);
-            type = market['type'];
+            [ type, params ] = this.handleMarketTypeAndParams ('fetchTickers', market, params);
```

Comment:
@AlgoQ here we don't need to do this because inside this if we're 100% sure that we have a symbol available so that's all we need to get the type